scalafmt-interfaces/src/main/java/org/scalafmt/interfaces/ScalafmtReporter.java

```diff
@@ -25,6 +25,18 @@
      */
     void error(Path file, Throwable e);
 
+    /**
+     * An exception occurred while trying to process this file.
+     *
+     * @param file can be either a Scala source file or .scalafmt.conf.
+     * @param message additional error message
+     * @param e the exception that occurred, has type {@link PositionException}
+     *          when the error appeared as a position.
+     */
+    default void error(Path file, String message, Throwable e) {
```

Comment:
Is this addition necessary? It seems we can introduce a new `MessageOnlyException` class instead to represent message errors. I don't feel strongly about it, just want to limit the surface of the interfaces.